CHANGELOG.md

```diff
@@ -2,25 +2,26 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-<a name="1.3.1"></a>
-## [1.3.1](https://github.com/Zooz/predator/compare/v1.3.0...v1.3.1) (2020-04-17)
-
-
 
 <a name="1.3.0"></a>
 # [1.3.0](https://github.com/Zooz/predator/compare/v1.2.1...v1.3.0) (2020-04-16)
 
 
-### Bug Fixes
-
-* **reports:** fixing grafana link for in progress tests ([#277](https://github.com/Zooz/predator/issues/277)) ([9236774](https://github.com/Zooz/predator/commit/9236774))
-
-
 ### Features
 
-* add score to mail and webhook ([#287](https://github.com/Zooz/predator/issues/287)) ([4f1e7c2](https://github.com/Zooz/predator/commit/4f1e7c2))
-* **reports:** add avg-rps to reports  ([#292](https://github.com/Zooz/predator/issues/292)) ([fa61a39](https://github.com/Zooz/predator/commit/fa61a39))
+* **reports** By creating a benchmark for a specific test, each subsequent test run for that test will be given a score from 0-100 summarizing the test run in one simple to analyze numerical value. ([#280](https://github.com/Zooz/predator/issues/280))
+* **reports** Compare multiple tests runs in the UI ([#290](https://github.com/Zooz/predator/issues/290))
+* **reports:** Reports will now show avg rps during all test instead of last rps stat  ([#292](https://github.com/Zooz/predator/issues/292)) ([fa61a39](https://github.com/Zooz/predator/commit/fa61a39))
+* **reports** Edit/Add notes from the reports/last reports screen ([#273](https://github.com/Zooz/predator/issues/273))
+* **metrics** Added support for setting custom labels for prometheus ([#278](https://github.com/Zooz/predator/issues/278))
+* **configuration** All configuration is now avaiable through the UI ([#274](https://github.com/Zooz/predator/issues/274))
+* **tests** New create/edit test form UI ([#271](https://github.com/Zooz/predator/issues/271)) 
+* **tests** Dynamic DSL. DSL tests will be transalted to predator tests dynamically ([#283](https://github.com/Zooz/predator/issues/283))
 
+### Bug Fixes
+
```

Comment:
```suggestion
**reports:** Reports will now show average RPS during the entire test run instead of the last RPS result  ([#292](https://github.com/Zooz/predator/issues/292)) ([fa61a39](https://github.com/Zooz/predator/commit/fa61a39))
```